ui/src/app/cohort-search/epics.ts

```diff
@@ -0,0 +1,53 @@
+import {ActionsObservable} from 'redux-observable';
```

Comment:
Should we have a directory for all the redux ts files? Like actions.ts, epics.ts, reducer.ts and store.ts